### ISSUE TITLE
Using directives for including video(s) into text

### ISSUE THREAD

**Issue by scheunemann** [ggueret/pelican-dashify#6]
Hello, 

A feature I would like to see would be the possibility of placing video(s) in an article or page with a directive or a tag. 

When using a directive, I would think it should look like:

```rst
.. dashify:: some-video.mp4
   :height: 100px
   :width: 200 px
   :alt: alternative text/or youtube link if video not loading
```
Ideally, this would play well with using links like `{static}/some-video.mp4` or `{attach}/some-video.mp4`. The latter is what I am interested in.

I was wondering if you considered extending the plugin with those capabilities? Or are you open for suggestions or PRs here?

**Comment by ggueret** [comment]
Yes! I would definitely like to see this feature into Pelican-dashify.
At a first glance, I've pushed back this to take the time to think about the generated html to a fair extent.

If you have any code to submit, a pull request is welcome. Otherwise, I would like to implement this new feature, it will be the opportunity to define the API scope.
